client/src/components/Flash/index.tsx

```diff
@@ -16,7 +16,8 @@ function Flash({ flashMessage, removeFlashMessage }: FlashProps): JSX.Element {
   const { type, message, id, variables } = flashMessage;
   const { t } = useTranslation();
 
-  const flashStyle = type as AlertProps['variant'];
+  const flashStyle =
+    type === 'error' ? 'danger' : (type as AlertProps['variant']);
```

Comment:
out of curiosity, what does that do? I have tested it in the alert PR, and I got blue, green and red alerts without it.

Reply:
It makes it so if it has a type of 'error' it sets it to 'danger' (red) - without it, it gives a white background (doesn't have anything for 'error'). I [added it here](https://github.com/freeCodeCamp/freeCodeCamp/pull/51551/files#diff-dbecd82f90a1904bbfffb280789ca1cf7b850db29c4ecfb65f8e1f589f6a4ab8R23) and it was [inadvertently removed here.](https://github.com/freeCodeCamp/freeCodeCamp/pull/51738/files#diff-dbecd82f90a1904bbfffb280789ca1cf7b850db29c4ecfb65f8e1f589f6a4ab8L23)